pkgs/tools/security/keybase/gui.nix

```diff
@@ -4,16 +4,16 @@
 , runtimeShell, gsettings-desktop-schemas }:
 
 let
-  versionSuffix = "20200320154633.3e235215b3";
+  versionSuffix = "20200424214931.7b0bbf1e3c";
 in
 
 stdenv.mkDerivation rec {
   pname = "keybase-gui";
-  version = "5.3.1"; # Find latest version from https://prerelease.keybase.io/deb/dists/stable/main/binary-amd64/Packages
+  version = "5.4.2"; # Find latest version from https://prerelease.keybase.io/deb/dists/stable/main/binary-amd64/Packages
 
   src = fetchurl {
     url = "https://s3.amazonaws.com/prerelease.keybase.io/linux_binaries/deb/keybase_${version + "-" + versionSuffix}_amd64.deb";
-    sha256 = "00k1xg27arbqpa836c55fhkxjvjbhjgkf6jqzprjzz76zksfdcx4";
+    sha256 = "fac7adf20e43e1a23d757bf7ac7a9011ee7a15388ddbc1faa735959473d5331a";
```

Comment:
Wait a minute - I don't think this is the right hash. How is this even working?

Reply:
The second one seems to be hex encoded, while the first is base32 I believe

Reply:
I pushed an amendment updating the encoding to base64, and I only now realize you wrote base **32**. How odd.

Reply:
Okay base32 now :)